packaging/windows/amazon-cloudwatch-agent-ctl.ps1

```diff
@@ -500,8 +511,24 @@ Function CWATestEC2() {
     return !$error
 }
 
-Function main() {
+Function SetLogLevelAll() {
+    switch -exact ($LogLevel) {
+        INFO { }
+        DEBUG { }
+        ERROR { }
+        WARN { }
+        OFF { }
+        default {
+            Write-Output "Invalid log level: ${LogLevel}`n${UsageString}"
+            Exit 1

```

Comment:
PowerShell syntax, I am familiar with:
```
Write-Output "Invalid log level: $LogLevel`n$UsageString"
```
(without extra braces around the vars).
What you have seems to work though:
https://code.labstack.com/powershell

Reply:
That's good to know. I just based it on other existing code within the script.